--- configure.in	2007-02-28 23:42:54.000000000 +0100
+++ configure.in.oden	2007-08-12 18:23:09.000000000 +0200
@@ -87,28 +87,20 @@
 AC_CHECK_LIB(cups, ppdOpenFile)
 AC_CHECK_LIB([cupsimage], [cupsRasterReadHeader])
 AC_CHECK_LIB([jbig], [jbg_enc_init],
-if test -f "/usr/lib64/libjbig.a"; then
-	LIBJBIG="/usr/lib64/libjbig.a"
-elif test -f "/usr/lib/libjbig.a"; then
-	LIBJBIG="/usr/lib/libjbig.a"
-elif test -f "/usr/local/lib64/libjbig.a"; then
-	LIBJBIG="/usr/local/lib64/libjbig.a"
-else
-	LIBJBIG="/usr/local/lib/libjbig.a"
+if test -f "/usr/lib64/libjbig.so"; then
+	LIBJBIG="-ljbig"
+elif test -f "/usr/lib/libjbig.so"; then
+	LIBJBIG="-ljbig"
 fi,
-[AC_MSG_ERROR([Could not find libjbig.a])])
+[AC_MSG_ERROR([Could not find -ljbig])])
 AC_SUBST(LIBJBIG)
-AC_CHECK_LIB([lcms], [cmsCreateTransform],
-if test -f "/usr/lib64/liblcms.a"; then
-	LIBLCMS="/usr/lib64/liblcms.a"
-elif test -f "/usr/lib/liblcms.a"; then
-	LIBLCMS="/usr/lib/liblcms.a"
-elif test -f "/usr/local/lib64/liblcms.a"; then
-	LIBLCMS="/usr/local/lib64/liblcms.a"
-else
-	LIBLCMS="/usr/local/lib/liblcms.a"
+AC_CHECK_LIB([lcms2], [cmsCreateTransform],
+if test -f "/usr/lib64/liblcms2.so"; then
+	LIBLCMS="-llcms2"
+elif test -f "/usr/lib/liblcms2.so"; then
+	LIBLCMS="-llcms2"
 fi,
-[AC_MSG_ERROR([Could not find liblcms.a])])
+[AC_MSG_ERROR([Could not find -llcms2])])
 AC_SUBST(LIBLCMS)
 
 # Checks for header files.
@@ -106,7 +106,7 @@ AC_SUBST(LIBLCMS)
 # Checks for header files.
 AC_HEADER_STDC
 AC_CHECK_HEADERS([fcntl.h limits.h stdlib.h string.h unistd.h])
-AC_CHECK_HEADERS([lcms.h icc34.h jbig.h])
+AC_CHECK_HEADERS([lcms2.h icc34.h jbig.h])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
