diff -up magicolor5440DL-1.2.1/src/rastertokmlf.c.omv~ magicolor5440DL-1.2.1/src/rastertokmlf.c
--- magicolor5440DL-1.2.1/src/rastertokmlf.c.omv~	2018-11-27 16:50:09.773215670 +0100
+++ magicolor5440DL-1.2.1/src/rastertokmlf.c	2018-11-27 16:50:18.421230525 +0100
@@ -53,6 +53,7 @@
  */
 
 #include <cups/cups.h>
+#include <cups/ppd.h>
 //#include <cups/string.h>
 #include <cups/language.h>
 #include <cups/raster.h>
