diff -up magicolor5440DL-1.2.1/configure.in.automake-1_13 magicolor5440DL-1.2.1/configure.in
--- magicolor5440DL-1.2.1/configure.in.automake-1_13	2013-01-03 23:49:39.996401062 +0100
+++ magicolor5440DL-1.2.1/configure.in	2013-01-03 23:50:23.973975922 +0100
@@ -4,7 +4,7 @@
 AC_PREREQ(2.57)
 AC_INIT(src/kmlf.h)
 AC_CONFIG_SRCDIR([src/rastertokmlf.c])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AM_INIT_AUTOMAKE(magicolor5440DL,1.2.1)
 # Checks for programs.
 AC_PROG_CC
